includes/payment-methods/class-wc-gateway-stripe-multibanco.php

```diff
@@ -194,7 +194,7 @@ public function payment_fields() {
 			data-currency="' . esc_attr( strtolower( get_woocommerce_currency() ) ) . '">';
 
 		if ( $description ) {
-			echo wpautop( esc_html( apply_filters( 'wc_stripe_description', wp_kses_post( $description ), $this->id ) ) );
+			echo wpautop( esc_html( apply_filters( 'wc_stripe_description', esc_html( $description ), $this->id ) ) );
```

Comment:
How about removing this first escaping to `$description` altogether? I know this was introduced previously, but it may be good to handle it now that we're around 🙂 

We have the `esc_html()` escaping the value returned by the `wc_stripe_description()` filter, so it'll be escaped before being printed. And double-escaping wouldn't be of much help.

```suggestion
			echo wpautop( esc_html( apply_filters( 'wc_stripe_description', $description, $this->id ) ) );
```

Reply:
TIL, thanks!